make.ps1

```diff
@@ -1,27 +1,5 @@
-function FindMSBuild
-{
-	$msBuildVersions = @("4.0")
-	foreach ($msBuildVersion in $msBuildVersions)
-	{
-		$key = "HKLM:\SOFTWARE\Microsoft\MSBuild\ToolsVersions\{0}" -f $msBuildVersion
-		$property = Get-ItemProperty $key -ErrorAction SilentlyContinue
-		if ($property -eq $null -or $property.MSBuildToolsPath -eq $null)
-		{
-			continue
-		}
-		$path = Join-Path $property.MSBuildToolsPath -ChildPath "MSBuild.exe"
-		if (Test-Path $path)
-		{
-			return $path
-		}
-	}
-	return $null
-}
-
-function UtilityNotFound
-{
-	echo "OpenRA.Utility.exe could not be found. Build the project first using the `"all`" command."
-}
+$functions = "$($MyInvocation.MyCommand.path | split-path)\make-functions.ps1" 
```

Comment:
I'm not convinced that the (minor, IMO?) readability improvement for shipping these into their own file outweighs the (minor) complexity of having a separate function file.  Wouldn't we get the same benefits by having all the functions at the top of the file?